flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/CheckpointStatsTrackerTest.java

```diff
@@ -461,6 +467,7 @@ public <T, G extends Gauge<T>> G gauge(String name, G gauge) {
         assertEquals(Long.valueOf(persistedData), latestPersistedData.getValue());
         assertEquals(Long.valueOf(ackTimestamp), latestCompletedDuration.getValue());
         assertEquals(externalPath, latestCompletedExternalPath.getValue());
+        assertEquals(Long.valueOf(0), latestCompletedId.getValue());
```

Comment:
I think we could also check the correctness of checkpointId when failed and in progress.

Reply:
Thanks for your tips , I have added check case

Reply:
Actually, we cannot set the checkpoint id as `0` in the production environment, I think we could set the pending checkpoint id as `1` by default.

Reply:
This is just a minor suggestion, it's fine to merge without such modification.